src/lib/youTubeApi.js

```diff
@@ -0,0 +1,17 @@
+const key = process.env.REACT_APP_API_KEY;
+
+export async function fetchVideos(keyWord) {
+  const apiBaseUrl = `https://content-youtube.googleapis.com/youtube/v3/search?part=id&part=snippet&maxResults=25&q=${keyWord}&key=${key}`;
+
+  return fetch(apiBaseUrl)
+    .then((response) => (response.ok ? response.json() : []))
+    .then((data) =>
+      data
+        ? {
+            items: data.items,
+          }
+        : {
+            items: [],
+          }
+    );
```

Comment:
You should catch errors and notify user properly, but at least catch the error here although you log the error only for the moment